### ISSUE TITLE
Two questions

### ISSUE THREAD

**Issue by Fpooyan** [KaihuaTang/Scene-Graph-Benchmark.pytorch#99]
## ❓ Questions and Help
Hi. Thanks for providing this great repo!

After I explored the code a bit, I faced two problems.
first, what is the difference between Predcl, SGCL, and SGDET in the training phase? do we finetune detector layers while training SGDET?
and the second question, while training Predcl using batch size=1 and base_lr=0.0025, the loss does not decrease and keeps oscillating. Does it have anything to do with the batch size?

Thanks for any help in advance,
